<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFichaBecado</class>
 <widget class="QMainWindow" name="QFichaBecado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ficha Becado</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lNombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombre">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lApellidos">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellidos *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eApellidos">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lNombrePadre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del Padre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombrePadre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lNombreMadre">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de la Madre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombreMadre">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lUrlFoto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>URL Foto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eUrlFoto">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaNacimiento">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de Nacimiento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaNacimiento">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaAltaAcoes">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de alta ACOES *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaAltaAcoes">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaAlta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de Alta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaAlta">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaSalidaAcoes">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de salida ACOES</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaSalidaAcoes">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lGrado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Grado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eGrado">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lColoniaNacimiento">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Colonia Nacimiento *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eColoniaNacimiento">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lColoniaResidencia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Colonia Residencia *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eColoniaResidencia">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lColegio">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Colegio *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eColegio">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lHistorial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Historial:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="eHistorial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>751</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lObservaciones">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Observaciones:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="eObservaciones">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>751</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bGuardarYSalir">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>600</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar y Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSalirSinGuardar">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir sin Guardar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lEstado">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eEstado">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
